.github/workflows/mcad-release.yml

```
@@ -41,3 +48,14 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ github.token }}
 
+    - name: Build
+      run: make mcad-controller
+
+    - name: Image Build and Push
+      env:
+        GIT_BRANCH: ${{ github.event.inputs.tag }}
+        TAG: release-${{ github.event.inputs.tag }}
+      run: |
+        make images -e GIT_BRANCH=$GIT_BRANCH TAG=$TAG
+        make push-images -e GIT_BRANCH=$GIT_BRANCH TAG=$TAG quay_repository=quay.io/${{ github.event.inputs.quay-organization }} quay_id=${{ secrets.QUAY_ID }} quay_token=${{ secrets.QUAY_TOKEN }}
+
```

```suggestion

    - name: Publish MCAD package
      run: |
        GOPROXY=proxy.golang.org go list -m github.com/project-codeflare/multi-cluster-app-dispatcher@${{ github.event.inputs.tag }}
```

Based on https://go.dev/doc/modules/publishing.
To make sure that the package is published and depending components can use it.

---

This is so cool!